BioSQL/BioSeq.py

```diff
@@ -191,7 +191,7 @@ def _retrieve_seq_len(adaptor, primary_id):
     if not seqs:
         return None
     assert len(seqs) == 1
-    given_length, = seqs[0]
+    (given_length,) = seqs[0]
```

Comment:
Why using a one-element tuple here? (If I understand this correctly).
Can't we use just `return int(seqs[0])`?

Reply:
I don't recall why the code was written that way in the first place, but it was me: 877a5f2dee468a160549f76b5fd91d65eea7ba29

There must be something unintuitive about the DB results object, the obvious fix fails:

```
TypeError: int() argument must be a string, a bytes-like object or a number, not 'tuple'
```

Reply:
There are more of these in ``Bio/File.py``, see e.g.

https://github.com/biopython/biopython/blob/biopython-175/Bio/File.py#L585

It seems to be an (anti) pattern with DB access.

Reply:
OK, now I got it, the contents of `seqs[0]` is a one-element tuple.

Reply:
Actually, I was irritated in the first place that tuple unpacking now also requires parenthesis. I don't like it, but OK.

Reply:
Ah - I was more bothered on why there was tuple unpacking (I wrote the code years ago, but this was non-obvious). I'm on the fence about the recent black style change - adding the brackets makes it difficult to overlook the tuple unpacking - previously that trailing comma was easy to miss.